arm-containerregistry/2017-03-01/examples/RegistryDelete.json

```diff
@@ -0,0 +1,12 @@
+{
+    "parameters": {
+        "api-version": "2017-03-01",
+        "subscriptionId": "subid",
+        "resourceGroupName": "rgname",
+        "registryName": "registry"
+    },
+    "responses": {
+        "200": {},
```

Comment:
Storage account will never be cleaned up by ACR. This is not our main line flow and no plan even in the future. 
This operation doesn't need to be Async.